src/main/java/gov/cms/ab2d/fetcher/FetcherHandler.java

```diff
@@ -0,0 +1,47 @@
+package gov.cms.ab2d.fetcher;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.amazonaws.services.lambda.runtime.events.KinesisEvent;
+import com.amazonaws.services.lambda.runtime.events.KinesisEvent.KinesisEventRecord;
+
+import java.io.IOException;
+
+/**
+ * The entry point invoked by AWS
+ */
+@SuppressWarnings("unused")
+public class FetcherHandler implements RequestHandler<KinesisEvent, String> {
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    @Override
+    public String handleRequest(KinesisEvent event, Context context) {
+        LambdaLogger logger = context.getLogger();
+
+        logger.log("Kinesis event version.");
+        String response = "200 OK";
+        for (KinesisEventRecord eventRecord : event.getRecords()) {
+            try {
+                JsonNode node = objectMapper.readTree(eventRecord.getKinesis().getData().array());
```

Comment:
Is there an advantage to using a JsonNode vs a dto here?

Reply:
It's more flexible, it is what is actually being sent.

I tend to start with JsonNode and only if DTOs are useful, do I transition to them.

Reply:
At some level, a JsonNode is a DTO.

Reply:
Ah ok I thought that was the case.